client/post-editor/editor-drawer/page-options.jsx

```diff
@@ -31,10 +31,7 @@ function EditorDrawerPageOptions( { translate, siteId, postId, post, postType, h
 	return (
 		<Accordion title={ title } icon={ <Gridicon icon="pages" /> }>
 			{ hierarchical && (
```

Comment:
Unrelated and for another pull request, but do you think we should make `<PageParent />` self-sufficient in deciding whether to render based on whether the post type is hierarchical (return `null` from its render if not)?